app/components/SearchResults/TracksResults/index.js

```diff
@@ -1,53 +1,45 @@
 import React from 'react';
-import artPlaceholder from '../../../../resources/media/art_placeholder.png';
+import _ from 'lodash';
 
-import styles from './styles.scss';
+import TrackRow from '../../TrackRow';
 
 class TracksResults extends React.Component {
   constructor(props) {
     super(props);
   }
 
-  renderTrack(track, index = 0) {
-    let addToQueue = this.props.addToQueue;
-    return (
-      <div
-        key={'track-' + index}
-        className={styles.track_row}
-        onClick={() => {
-          addToQueue(this.props.musicSources, {
-            artist: track.artist,
-            name: track.name,
-            thumbnail: track.image[1]['#text'],
-          });
-        }}
-      >
-        <img src={track.image[0]['#text'] || artPlaceholder} />
-        <div className={styles.popular_track_name}>
-          {track.artist} - {track.name}
-        </div>
-      </div>
-    );
-  }
-
-  render() {
+  render () {
     let collection = this.props.tracks || [];
     let limit = this.props.limit;
-    if (collection.length == 0) {
+    if (collection.length === 0) {
       return 'No result';
     } else {
       return (
-        <div className={styles.popular_tracks_container}>
-          {(collection || []).slice(0, limit).map((track, i) => {
-            if (track) {
-              if (track.name && track.image && track.artist) {
-                {
-                  return this.renderTrack(track, i);
+        <table>
+          <tbody>
+            {(collection || []).slice(0, limit).map((track, index) => {
+              if (track && _.hasIn(track, 'name') & _.hasIn(track, 'image') && _.hasIn(track, 'artist')) {
+                if (!track.artist.name) {
+                  track.artist = { name: track.artist };
```

Comment:
As above, this will mutate the props. Better to use `Object.assign`, e.g. `let trackCopy = Object.assign({}, track, {artist: {name: track.artist}})`, this way you get a clone of the object.